LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Marvin Menzerath
```

Comment:
Would you kindly change `2016` to `2014-2016` to reflect the long history of this project?